### PR TITLE
[MIG] stock_picking_import_serial_number: Migration to 14.0

### PR DESCRIPTION
**Pull request by ryanc-me**
No changes in logic. The only addition is `ir.model.access.csv`, which is now required for transient models.

### CODE REVIEW

File: stock_picking_import_serial_number/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Import Serial Numbers",
     "summary": "Import S/N from excel file for incoming pickings",
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.2",
```
Comment (sebalix) [inline stock_picking_import_serial_number/__manifest__.py]: You can reset the version number to `14.0.1.0.0`

Comment (ryanc-me) [reply stock_picking_import_serial_number/__manifest__.py]: Thanks @sebalix. Do you know if OCA accepts amend & force-push?

Comment (pedrobaeza) [reply stock_picking_import_serial_number/__manifest__.py]: Yes

**Comment by rousseldenis** [comment]
@ryanc-me

**Comment by ryanc-me** [comment]
@sebalix Ready to go 👍

**Review by sebalix — APPROVED** [review]

**Comment by ryanc-me** [comment]
@sebalix @rousseldenis could you guys please re-review?

This is a 14.0 migration of the current 13.0 version.

Note that there is a [known edge-case bug](https://github.com/OCA/stock-logistics-workflow/pull/1062) in that version, with an in-progress PR #1065.

@pedrobaeza is it okay to migrate as-is like this? Once #1065 is merged I can create a PR to forward-port it (unless that's handled by OCA in the background).

**Comment by pedrobaeza** [comment]
Yes, it's reasonable.

**Pull request merged**